charts/opentelemetry-operator/templates/deployment.yaml

```diff
@@ -32,6 +32,11 @@ spec:
             {{- end }}
           command:
             - /manager
+          {{- if not (and .Values.admissionWebhooks.enabled .Values.admissionWebhooks.certManager.enabled) }}
```

Comment:
Shouldn't it be `{{- if .not Values.admissionWebhooks.enabled }}`? Is it valid/useful configuration when `ENABLE_WEBHOOKS` isn't set to `false` but `.Values.admissionWebhooks.enabled=true and .Values.admissionWebhooks.certManager.enabled=false`?

Reply:
So certmanager is the main dependency for to secrets volume to be created/mounted and the pod to start with webhooks enabled. So if we want to disable the webhooks we need all 3 flags disabled. I mean we could have certmanager enabled but I don't see the point as it has only this function. 
Anyway, give me some more time as the current pr can have a duplicate `env` key which I need to fix.

Reply:
But to answer your question yes there is a valid use case for what you said: webhooks enabled and the tls secret provided manually without certmanager.

Reply:
@dmitryax I think this is the best version so far. Problem is that it introduces a breaking change for current users that have env vars defined.
Let me know your thoughts.

Reply:
I don't think it's going to be a significant breaking change if we don't introduce `env.enabled`.

Reply:
Otherwise, the PR is in na good shape